common/scripts/get_source.sh

```diff
@@ -30,6 +30,11 @@
 
 TOP_DIR=`pwd`
 
+#Optional argument 'arm' shall be set when targeting a 32bit Arm device
```

Comment:
Initialise 
TARGET_ARCH="aarch64"

else the script will give error in line 175 check as $TARGET_ARCH is NULL.

  if [ $TARGET_ARCH != "arm" ]; then

Reply:
done